frontend/src/Admin/components/CreateShowForm.js

```diff
@@ -1,120 +1,216 @@
+          handleSubmit,
+          isSubmitting
+        }) => (
+          <Form onSubmit={handleSubmit}>
             <Row>
-              <Col>
+              <Col xs={6}>
                 <FormGroup>
-                  <Label>Submission Dates</Label>
-                  <CalendarContainer>
-                    {/* TODO: Update the selected dates if they're manually typed */}
-                    <Fields
-                      names={['entryStart', 'entryEnd']}
-                      component={props => (
-                        <DateRangePicker
-                          startDateFieldName='entryStart'
-                          endDateFieldName='entryEnd'
-                          className='form-control'
-                          {...props}
-                        />
-                      )}
-                    />
-                  </CalendarContainer>
+                  <Label>Name</Label>
+                  <Field
+                    type='text'
+                    id='name'
+                    name='name'
+                    className='form-control'
+                    required
+                  />
+                  {this.renderErrors(touched, errors, 'name')}
                 </FormGroup>
-              </Col>
-            </Row>
-            <Row>
-              <Col>
                 <FormGroup>
-                  <Label>Judging Dates</Label>
-                  <CalendarContainer>
-                    <Fields
-                      names={['judgingStart', 'judgingEnd']}
-                      component={props => (
-                        <DateRangePicker
-                          startDateFieldName='judgingStart'
-                          endDateFieldName='judgingEnd'
-                          {...props}
-                        />
-                      )}
-                    />
-                  </CalendarContainer>
+                  <Label>Description</Label>
+                  <Field
+                    component='textarea'
+                    id='description'
+                    name='description'
+                    className='form-control'
+                    rows={3}
+                  />
+                  {this.renderErrors(touched, errors, 'description')}
+                </FormGroup>
+                <FormGroup>
+                  <Label>Individual Submission Limit</Label>
+                  <Field
+                    type='number'
+                    min='1'
+                    step='1'
+                    id='entryCap'
+                    name='entryCap'
+                    className='form-control'
+                    required
+                  />
+                  {this.renderErrors(touched, errors, 'entryCap')}
                 </FormGroup>
               </Col>
+              <Col xs={6}>
+                <Row>
+                  <Col>
+                    <FormGroup>
+                      <Label>Submission Dates</Label>
+                      <CalendarContainer>
```

Comment:
Is there a simple way to allow time input on this form too? When the show starts and ends on the same day it doesn't work (I know this has been an issue, but this seems like a good time to fix it if it's a simple change).

Reply:
Simple – no. Possible – probably.

Yes, the validation on the frontend requires the day after. I think the validation on the backend only requires greater than time. 

We can double check what MySQL defaults the time to (or set it ourselves in our API) and indicate that in a tooltip on this page – we'll have to show it to students, too, so they don't miss the deadline.

Reply:
I think we need to handle time before we hand this off to Nanette, but not handling it in this change is fine.